problem/0000-ci-health-metrics.md

```diff
@@ -0,0 +1,68 @@
+- Contribution Name: ci-health-metrics
+- Implementation Owner: @alpeb, @kleimkuhler
+- Start Date: 2020-03-16
+- Target Date: 2020-04-03
+- RFC PR: [linkerd/rfc#11](https://github.com/linkerd/rfc/pull/11)
+- Linkerd Issue:
+  [linkerd/linkerd2#4176](https://github.com/linkerd/linkerd2/issues/4176)
+- Reviewers: @grampelberg
+
+# Summary
+
+[summary]: #summary
+
+This RFC proposes a service that would monitor the results from each CI run. It
+should allow us to catch consistently flaky tests and have an overview of the CI
+in terms of success rates and time performance.
+
+# Problem Statement (Step 1)
+
+[problem-statement]: #problem-statement
+
+CI jobs inevitably fail occasionally, either because of real problems with the
+code, or because of test flakiness. We'd like to stay informed about what tests
+are being persistently flaky, without having to dig into the CI run results
+themselves. This will allow us to properly prioritize work to fix those
+failures.
+
+Besides identifying the most common failures, by surfacing error metrics from the
+CI jobs we'll also have at our disposal an overview of CI's performance, in
+terms of things like success rates and time taken by the jobs. This information
+can be aggregated at the global, workflow and job levels.
```

Comment:
This is a somewhat handwavey description of the metrics that we wish to collect. Can we get more specific? I.e., independent of implementation, what do we want to measure?

Reply:
Ok, I've clarified in my last push what is that we want to measure.